apps/glusterfs/app.go

```diff
@@ -326,6 +331,11 @@ func (a *App) setBlockSettings() {
 		// Should be in GB as this is input for block hosting volume create
 		BlockHostingVolumeSize = a.conf.BlockHostingVolumeSize
 	}
+	if a.conf.BlockHostingVolumeOptions != "" {
```

Comment:
I'm happy to see this line because we don't want to force users to update the config for a new default  (we've made that mistake already). 
However, I do wonder how users are supposed to "unset" an option like the block group. Would you just suggest to set a "dummy" option?

Reply:
Yes, in most cases they should not unset block group. In one off cases, they must use skip option per volume create. If they do want to unset from default setting, they would have to define some dummy option say "user.rtalur.dummyoption true"